migrations/versions/0e80c49a6e28_.py

```diff
@@ -0,0 +1,29 @@
+"""empty message
+
+Revision ID: 0e80c49a6e28
+Revises: 6f7b6fad3f54
+Create Date: 2019-05-20 21:56:44.770504
+
+"""
+
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '0e80c49a6e28'
+down_revision = '6f7b6fad3f54'
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('events', sa.Column('is_ticket_form_enabled', sa.Boolean(), server_default='False', nullable=False))
+    op.add_column('events_version', sa.Column('is_ticket_form_enabled', sa.Boolean(), server_default='False', autoincrement=False, nullable=True))
```

Comment:
@uds5501 don't you think `autoincrement=False` should be removed and `nullable` should be false here?